audience_manager/authorized_api/definitions/profile_details.yaml

```diff
@@ -32,11 +32,11 @@ allOf:
             description: Number of conversions.
             example: 4
           last_revenue:
-            type: float
+            type: number
```

Comment:
can it be
```suggestion
            type: number
            format: float
```
?

Reply:
If it should be a float then OK. https://swagger.io/docs/specification/data-models/data-types/#numbers